apps/api/src/app.config.ts

```diff
@@ -1,7 +1,7 @@
 import { NestFactory } from '@nestjs/core';
 import * as winston from 'winston';
 import { WinstonModule, utilities as nestWinstonModuleUtilities } from 'nest-winston';
-import { Logger, ValidationPipe } from '@nestjs/common';
+import { Logger } from '@nestjs/common';
```

Comment:
Stray change here

Reply:
Validation pipe was an unused import. I figured I would remove it when I saw it.

Reply:
Oh sorry about that, I read this as an addition 👍